regparser/grammar/terms.py

```diff
@@ -26,10 +26,15 @@
     LineStart()
     + Optional(Suppress(unified.any_depth_p))
     + e_tag.setResultsName("head")
+    + Suppress(ZeroOrMore(unified.any_depth_p))
     + ZeroOrMore(
         (atomic.conj_phrases + e_tag).setResultsName(
             "tail", listAllMatches=True))
+    + Suppress(ZeroOrMore(Regex(r",[a-zA-Z ]+,")))
```

Comment:
Oh, I see the commas now! Okay, so these are comma clauses. Interesting! May or may not be helpful to spell that out for slow eyes like mine.